packages/ui/src/features/input-model/input-model.tsx

```diff
@@ -37,11 +37,11 @@ type InputModelProps = {
     newVariable: CategoricalVariableType
   ) => void
   setCategoricalVariableEnabled?: (index: number, enabled: boolean) => void
-  violations?: ValidationViolations
 }
 
 export function InputModel(props: InputModelProps) {
   const {
+    id,
```

Comment:
It could be useful if all reusable views have a default id assigned, e.g.,

```suggestion
    id='inputModel',
```

Reply:
Added